Documentation/Metrics/arangodb_request_body_size.yaml

```diff
@@ -0,0 +1,15 @@
+name: arangodb_request_body_size
```

Comment:
We should rename this to `arangodb_request_body_memory_usage`, in order to avoid confusion with the already existing metrics `arangodb_request_body_size_vst`, `arangodb_request_body_size_http1`, `arangodb_request_body_size_http2`.